modules/data-table/main/index.coffee

```diff
@@ -27,6 +34,25 @@ columnOptionLookup = (options, name, id) ->
   else
     options[name]
 
+toCriteriaItems = (list) ->
+  hx.unique(list).map (item) ->
+    {
+      value: item,
+      text: hx.userFacingText('dataTable', item)
+    }
+
+advancedSearchCriteriaValidate = (value) ->
+  allowedTypes = hx.filter.types()
+  if (hx.isArray(value) and value.every((c) -> ~allowedTypes.indexOf(c))) or value is undefined
```

Comment:
Can this be changed to `>=0` or `!==-1` instead of tilde? More readable in my opinion